modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/utils/odata.es.js

```diff
@@ -80,7 +81,22 @@ function buildQueryString(criteria, queryConjunction) {
 				}
 				else if (isValueType(FUNCTIONAL_OPERATORS, operatorName)) {
 					queryString = queryString.concat(
-						`${operatorName} (${propertyName}, '${value}')`
+						`${operatorName}(${propertyName}, '${value}')`
+					);
+				}
+				else if (isValueType(NOT_OPERATORS, operatorName)) {
+					const baseOperator = operatorName.replace(/not-/g, '');
+
+					const baseExpression = [{
+						operatorName: baseOperator,
+						propertyName,
+						value
+					}];
+
+					// Not is wrapped in a group to simplify AST parsing.
```

Comment:
Is this quote for line 98? If so, lets remove the new line between the two, otherwise this comment looks like its just floating out there. Unless this is a linting rule.

Reply:
Ya linting rule requires a new line above and below a comment. 😕